getdata.py

```diff
@@ -1,30 +1,33 @@
 import requests
-from pathlib import Path
 import xml.etree.ElementTree as ET
-import os
```

Comment:
Achei válido a alteração proposta, porque executa a ação com uma única biblioteca ao invés de duas, por mais que ambas sejam bult-in, assim se reduz a dependência para rodar o código.